index.bs

```diff
@@ -20,6 +20,12 @@ spec:fetch; type:dfn; text:client
 spec:ServiceWorker; type:dfn; for:/; text:
 spec:url; type:dfn; for:url; text:origin
 spec:html; type:element; text:style
+spec:infra; type:dfn; text:user agent
+</pre>
+
+<pre class="anchors">
+spec: HTML; urlPrefix: https://html.spec.whatwg.org/multipage/browsing-the-web.html
+    type: dfn; text: process a navigate response; url: process-a-navigate-response
```

Comment:
Should we point to "populating a session history entry" instead? Or is that the bit that gives you a linker error?

Reply:
You're right, that's a cleaner link since the other text isn't in the doc anymore